docs/deployment/production.md

```diff
@@ -14,20 +14,29 @@ We use nginx as a simple web server.
 
 ## Github Actions
 
-### Configuration
+There are two workflows used to test and deploy the Design System website.
 
-The Github Actions CI workflow is configured in [.github/workflows/ci.yaml](../../.github/workflows/ci.yaml).
-It will attempt to build the Design System, run the linter and run tests for every branch,
-but will only run the deploy step when on the main branch.
+### Testing
 
-Most of the environment variables needed are set within each Github Action script. Any secrets can be viewed and changed within the Github UI (requires admin access).
+The test workflow is configured in [.github/workflows/test.yaml](/.github/workflows/test.yaml).
+
+This workflow is set to run on every branch except main, where it is instead run as a dependency on the deployment workflow.
+
+The workflow will ensure the Design System website can be built successfully, then run the linter and the tests.
 
 ### Deployment
 
-The Github Actions deployment script is detailed in [.github/workflows/ci.yaml](../../.github/workflows/ci.yaml)
+The deployment workflow is configured in [.github/workflows/deploy.yaml](/.
+github/workflows/deploy.yaml).
```

Comment:
I think the GitHub Markdown renderer doesn't like this line break? When I look at the preview the link doesn't appear as expected.

```suggestion
The deployment workflow is configured in [.github/workflows/deploy.yaml](/.github/workflows/deploy.yaml).
```